README.md

```diff
@@ -140,6 +294,11 @@ const formatError = function (error) {
 app.use('/graphql', bodyParser.json(), graphqlExpress({ schema, formatError }))
 
 ```
+
+Apollo server plugin throws [`UserInputError`](https://www.apollographql.com/docs/apollo-server/data/errors/#bad_user_input) for each validatin error.
+Envelop plugin throws prefilled `GraphQLError` for each validation error.
+
 ### uniqueTypeName
 ```@constraint(uniqueTypeName: "Unique_Type_Name")```
-Override the unique type name generate by the library to the one passed as an argument
+Override the unique type name generate by the library to the one passed as an argument. 
+Has meaning only for `Schema wrapper` implementation.
```

Comment:
Can we provide more context here? I.e. what is schema wrapper, e.g. Apollo etc

Reply:
Schema wrapper based implementation is described above. I don't think it is necessary to repeat it here.